src/components/renderer/LookupNotification.tsx

```diff
@@ -362,6 +367,25 @@ async function fetchLookup(
     throw new Error('Could not find element lookup configuration')
   }
 
+  if (formElementLookup.type === 'STATIC_DATA') {
+    const elementName = payload.element.name
+    const inputValue = payload.submission[elementName]
+    const records = formElementLookup.records || []
+    const matchingRecord = records.find((r) => r.inputValue === inputValue)
+    const lookupResult: FormSubmissionModel = {}
+
+    if (!matchingRecord) {
+      return lookupResult
+    }
+
+    matchingRecord.preFills.forEach((prefill) => {
+      if (prefill.type === 'TEXT') {
+        lookupResult[prefill.formElementName] = prefill.text
+      }
+    })
+    return lookupResult
+  }
```

Comment:
This needs to go above the `if (!formElementLookup || !formElementLookup.url) {` check, otherwise it will never get to this point.